search-enterprise/latest/using/index.md

```diff
@@ -230,6 +230,28 @@ In the REST API you can specify the timezone to be used in search for date range
 }
 ```
 
+### Search using date math
+
+Date range queries can be even more powerful when applying date math functions. AFTS supports adding and subtracting periods, as well as rounding:
+
+| AFTS query | Description | Notes |
+| ---------- | ----------- | ----- |
+| `acme:projectStartDate:[NOW TO NOW+1DAY>` | Documents with a project start date in the next twenty four hours | |
+| `acme:projectStartDate:[NOW/DAY TO NOW/DAY+1DAY>` | Documents with a project start date today (between midnight and midnight UTC) | Note the subtle difference with the previous query |
```

Comment:
Hello @tpage-alfresco thanks for this. When you say between midnight and midnight UTC, what is the timezone of the first midnight?

Cheers

Reply:
Also UTC: "between (midnight and midnight) UTC"